scripts/rollup/wrappers.js

```diff
@@ -294,6 +298,55 @@ ${license}
  * ${'@gen' + 'erated'}
  */
 
+${source}`;
+  },
+
+  [IIFE_DEV](source, globalName, filename, moduleType) {
```

Comment:
We can skip this. Remember these files are going to get sent straight to the browser with no post-processing by a bundler or minifier. Don't want the extra bytes.